rocks-lib/src/build/builtin.rs

```diff
@@ -30,10 +32,11 @@ impl Build for BuiltinBuildSpec {
         progress: &Progress<ProgressBar>,
     ) -> Result<(), Self::Err> {
         // Detect all Lua modules
-        let modules = autodetect_modules(build_dir)
-            .into_iter()
-            .chain(self.modules)
-            .collect::<HashMap<_, _>>();
+        let modules = if self.modules.is_empty() {
```

Comment:
question(non-blocking): is this luarocks's behaviour too? Is there no situation where a few modules are defined by the rockspec author and they "expect" the rest to be autodetected? Given the amount of wacky code we've seen it might be worth checking haha

Reply:
I'm not sure. 
I tried installing `luassert` and it failed if using both the rockspec modules and the auto-detected modules.

Reply:
Done. Excluding paths from the `ModuleSpec` seems to solve the problem.